test/unit/service-cov-notify.spec.ts

```diff
@@ -3,12 +3,13 @@ import assert from 'node:assert'
 
 import * as utils from './utils'
 import { CovNotify } from '../../src/lib/services'
+import { ZERO_DATE } from '../../src/lib/asn1'
 
 test.describe('bacnet - Services layer COVNotify unit', () => {
 	test('should successfully encode and decode', () => {
 		const buffer = utils.getBuffer()
-		const date = new Date(1, 1, 1)
-		const time = new Date(1, 1, 1)
+		const date = ZERO_DATE
```

Comment:
This is still in draft because this should have broken these tests (`date` and `time` are the same object!) but apparently did not.